hnsecure/src/main/java/ca/bc/gov/hlth/hnsecure/json/fhir/ProcessV2ToJson.java

```diff
@@ -0,0 +1,27 @@
+package ca.bc.gov.hlth.hnsecure.json.fhir;
+
+import static ca.bc.gov.hlth.hnsecure.message.ErrorMessage.HL7Error_Msg_NoInputHL7;
+
+import org.apache.camel.Handler;
+import org.eclipse.jetty.util.StringUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import ca.bc.gov.hlth.hncommon.json.fhir.FHIRJsonUtil;
+import ca.bc.gov.hlth.hnsecure.exception.CustomHNSException;
+
+public class ProcessV2ToJson {
+	
+	private static final Logger logger = LoggerFactory.getLogger(ProcessV2ToJson.class);
+	
+	@Handler
+	public String convertToFHIRJson(String base64V2Message) throws CustomHNSException {
+		logger.debug("process: Trying to create a JSON Message {}", base64V2Message);
```

Comment:
Maybe update the logging to include the method name like I did in the HNClient version of this class.

Reply:
Done